cmd/backup.go

```diff
@@ -87,3 +96,17 @@ func newTableBackupCommand() *cobra.Command {
 	task.DefineTableFlags(command)
 	return command
 }
+
+// newRawBackupCommand return a raw kv range backup subcommand.
+func newRawBackupCommand() *cobra.Command {
+	command := &cobra.Command{
+		Use:   "raw",
+		Short: "backup a raw kv range from TiKV cluster",
+		RunE: func(command *cobra.Command, _ []string) error {
+			return runBackupRawCommand(command, "Raw Backup")
```

Comment:
```suggestion
			return runBackupRawCommand(command, "Raw backup")
```
Keep consistent with other commands

Reply:
Done